pydarn/plotting/rtp.py

```diff
@@ -56,7 +56,8 @@ def plot_range_time(cls, dmap_data: List[dict], parameter: str = 'v',
                         start_time: datetime = None, end_time: datetime = None,
                         colorbar: plt.colorbar = None, ymin: int = None,
                         ymax: int = None, yspacing: int = 200,
-                        coord: object = Coord.SLANT_RANGE, colorbar_label: str = '',
+                        coord: object = Coords.SLANT_RANGE, hgt = 250, 
```

Comment:
``` suggestion
                        coord: object = Coords.SLANT_RANGE, reflection_height = 250, 
```
Strongly advise not using abbreviations if the word is simple enough, try to make it descriptive following PEP 8 standards.